website/screens/components/wizard/usage/examples/content.ts

```diff
@@ -0,0 +1,40 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    const userIcon = (
```

Comment:
what do you think about moving this icon to an external file to have a cleaner code?